src/apps/backend/modules/task/rest-api/task-router.ts

```diff
@@ -15,5 +15,6 @@ export default class TaskRouter extends ApplicationRouter {
     router.get('/:id', ctrl.getTask);
     router.patch('/:id', ctrl.updateTask);
     router.delete('/:id', ctrl.deleteTask);
+    router.post('/share', ctrl.shareTask);
```

Comment:
this isn't a traditional `CRUD` api

Also, I don't think we need a separate api for this. Since we are just updating the document's existing field, we can do that from the `patch` api itself.